applications/common/frontend/src/helpers/breakpoints.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 2022 Dynamic Solutions
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const breakpoints = {
```

Comment:
We have already some breakpoints defined in https://github.com/websight-io/kyanite/blob/main/applications/common/frontend/src/atomic-design-system/00-token/t.breakpoint/t.breakpoint.scss

Reply:
It's not possible to reuse SASS or CSS variables in JS. In Howlite we also define the same breakpoints in JS.

Reply:
Right, I missed it's from CSS